scripts/utils/unist-fs-builder.js

```diff
@@ -0,0 +1,23 @@
+const u = require('unist-builder');
+const path = require('path');
+
+const directory = (dirpath, children = []) => {
+  return u(
+    'directory',
+    { basename: path.basename(dirpath), path: dirpath },
+    children
+  );
+};
+
+const file = (filepath, data) => {
```

Comment:
What is the purpose of adding these as a syntax tree?

Reply:
I was able to leverage the utilities in the [`unist`](https://github.com/syntax-tree/unist#utilities) suite of utilities to traverse the directory tree, rather than having to implement the traversal myself. 

It does bend the rules just a bit since these are technically utilities for a "syntax tree", but they worked pretty well for representing the file system as well. Figured I might as well leverage a bunch of tooling that already existed to avoid having to reimplement something very similar. Does that answer your question?